splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/MergeSortJoinOperation.java

```diff
@@ -198,7 +198,7 @@ public String prettyPrint(int indentLevel) {
             SpliceLogUtils.debug(LOG, "getDataSet Performing MergeSortJoin type=%s, antiJoin=%s, hasRestriction=%s",
                     getJoinTypeString(), isAntiJoin(), restriction != null);
         }
-        rightDataSet1.map(new CountJoinedRightFunction(operationContext));
+        //rightDataSet1.map(new CountJoinedRightFunction(operationContext));
```

Comment:
What's the reason for commenting this out?

Reply:
That was a left-over from some debugging session. Thanks for catching it, I will remove it and push a new commit.